src/app/features/reporting/reporting/reporting.component.html

```diff
@@ -19,7 +19,7 @@
 
 <div style="color: gray; padding: 10px" *ngIf="reportCalculation" i18n>
   This report was calculated at:
-  {{ reportCalculation?.endDate | date: "dd.MM.yyyy HH:mm" }} <br />
+  {{ localTime | date: "dd.MM.yyyy HH:mm" }} <br />
```

Comment:
You solution seems to work well. Doing a quick research if we can avoid the extra property, I noticed the following:

```JavaScript
new Date("2025-02-18T08:05:45.909")
// = Tue Feb 18 2025 08:05:45 GMT+0100 (Central European Standard Time)
new Date("2025-02-18T08:05:45.909" + "Z")
// = Tue Feb 18 2025 09:05:45 GMT+0100 (Central European Standard Time)
```

i.e. probably our backend should write the date including "Z" to indicate UTC timezone.

Maybe we can just add that in `fetchReportCalculation()` service as a workaround, without having to edit the component here?

e.g. 
```
this.http.get<ReportCalculation>(...)
  .pipe(map(res => {
    if(res?.endDate && !res.endDate.endsWith("Z")) {
      res.endDate = res.endDate + "Z";
    }
    return res;
  })`

Reply:
nvermind. The proposed solution is working and not too complex anyway :sweat_smile: